portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -69,12 +69,62 @@ definition {
 	}
 
 	@description = "Verify that a Form can be submitted with the Paragraph Field"
-	@ignore = "true"
 	@priority = "5"
 	test CanBeSubmitted {
 		property portal.acceptance = "false";
 
-		// TODO LPS-131229 CanBeSubmitted pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Paragraph");
+
+		var fieldParagraphField = FormFields.getFieldName();
```

Comment:
You are declaring a var here but it is not being referenced elsewhere. Is it possible to take this line out and have the test still pass?

Reply:
Yes, I pushed the test with this change